.pre-commit-config.yaml

```
@@ -779,7 +779,7 @@ repos:
         files: ^\.pre-commit-config\.yaml$|^.*_installed_providers\.txt$
         pass_filenames: false
         require_serial: true
-      - id: update-spelling-wordlist-to-be-sorted
+      - id: update--wordlist-to-be-sorted
```

```suggestion
      - id: update-spelling-wordlist-to-be-sorted
```
we probably don't want to change it. i guess this is an accident

---

I have pushed another update with the `pre-commit` changes.